FiveCalls/FiveCalls/ScheduleRemindersController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ScheduleRemindersController.swift
+//  FiveCalls
+//
+//  Created by Christopher Brandow on 2/8/17.
+//  Copyright © 2017 5calls. All rights reserved.
+//
+
+import Foundation
+
+class ScheduleRemindersController: UIViewController, MultipleSelectionDelegate {
+
+    @IBOutlet weak var timePicker: UIDatePicker!
+    @IBOutlet weak var daysOfWeekSelector: MultipleSelectionControl!
+
+    var notificationsChanged: Bool?
+
+    override func viewDidLoad() {
+        daysOfWeekSelector.delegate = self
+        notificationsChanged = false
+        if let notifications = UIApplication.shared.scheduledLocalNotifications {
+            daysOfWeekSelector.setSelectedButtons(at: indices(from: notifications))
+            if let date = notifications.first?.fireDate {
+                timePicker.setDate(date, animated: true)
+            }
+        }
+        timePicker.setValue(UIColor(red:0.12, green:0.47, blue:0.81, alpha:1.00), forKey: "textColor")
```

Comment:
heads up there is a PR to move all these colors into static properties, so we'll have to change this once that is merged in